src/seedu/addressbook/data/AddressBook.java

```diff
@@ -99,6 +99,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Sorts all persons in the address book.
+     */
+    public void sortPersons() {
```

Comment:
Does it change the order of data stored in the database? Is it possible that after "sort" user uses that index to delete and unintended person may be deleted?

Reply:
You should specify here the sequence order in which the list of persons are sorted: i.e : name, address, email

Reply:
Well done on adding a sort function, however remember to update User Guide as well as the help command :)